Sources/MapboxNavigation/FeedbackSubtypeViewController.swift

```diff
@@ -29,6 +30,10 @@ class FeedbackSubtypeViewController: FeedbackViewController {
 
         updateButtonTitle()
     }
+    
+    override func createFeedback() {
+        // do nothing, feedbackUUID should be passed to the initializer
```

Comment:
NIT: It's just `feedback`, not `feedbackUUID`

Reply:
Thanks!